### DISCUSSION TITLE
Deployment within Github

### DISCUSSION THREAD

**Discussion by Kalithasan-Periyasamy** [github/safe-settings#569] [Q&A] [answered]
Is this possible to deploy this app and run with in github ?

**Comment by svg153** [comment]
As far as I know, it is not possible for now.  If the app is modified so that by command line using a flag it can be started with the cron logic. But it would have to be investigated.

**Reply by Kalithasan-Periyasamy** [reply, marked as answer]
Okay. Thanks @svg153.
